.circleci/config.yml

```diff
@@ -170,7 +170,7 @@ jobs:
       - run:
           command: |
             # See https://circleci.com/docs/2.0/env-vars/#setting-an-environment-variable-in-a-shell-command
-            TAG=$CIRCLE_SHA1 docker-compose -f ./docker/compose/docker-compose.yml up -d
+            TAG=$CIRCLE_SHA1 docker-compose -f ./docker/compose/docker-compose.test.yml up -d
```

Comment:
perhaps we should move this top-level, next to docker-compose.test.yml. We could reserve `./docker/compose/` for public examples?